README.md

```diff
@@ -135,11 +135,12 @@ module "aws_config" {
 | check\_vpc\_default\_security\_group\_closed | Enable vpc-default-security-group-closed rule | `bool` | `true` | no |
 | config\_aggregator\_name | The name of the aggregator. | `string` | `"organization"` | no |
 | config\_delivery\_frequency | The frequency with which AWS Config delivers configuration snapshots. | `string` | `"Six_Hours"` | no |
-| config\_logs\_bucket | The S3 bucket for AWS Config logs. | `string` | n/a | yes |
+| config\_logs\_bucket | The S3 bucket for AWS Config logs. If you have set enable\_config\_recorder to false then this can be an empty string. | `string` | n/a | yes |
```

Comment:
I guess I'm confused why this is required if it can be left empty? Wouldn't it be easier to just make the empty string the default? I guess you probably run into an error if you have the recorder turned on and the bucket is empty, though?

Reply:
NEVERMIND, I should read the rest of the stupid PR first...